specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2024-05-01/appconfiguration.json

```diff
@@ -0,0 +1,2890 @@
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "status": {
+          "description": "The current status of the snapshot.",
+          "enum": [
+            "Provisioning",
+            "Ready",
+            "Archived",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "SnapshotStatus",
+            "modelAsString": true
+          }
+        },
+        "filters": {
+          "description": "A list of filters used to filter the key-values included in the snapshot.",
+          "type": "array",
+          "minItems": 1,
+          "maxItems": 3,
+          "items": {
+            "$ref": "#/definitions/KeyValueFilter"
+          },
+          "x-ms-identifiers": []
+        },
+        "compositionType": {
+          "description": "The composition type describes how the key-values within the snapshot are composed. The 'key' composition type ensures there are no two key-values containing the same key. The 'key_label' composition type ensures there are no two key-values containing the same key and label.",
+          "type": "string",
+          "enum": [
+            "Key",
+            "Key_Label"
+          ],
+          "x-ms-enum": {
+            "name": "CompositionType",
+            "modelAsString": true
+          }
+        },
+        "created": {
+          "description": "The time that the snapshot was created.",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "expires": {
+          "description": "The time that the snapshot will expire.",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "retentionPeriod": {
+          "description": "The amount of time, in seconds, that a snapshot will remain in the archived state before expiring. This property is only writable during the creation of a snapshot. If not specified, the default lifetime of key-value revisions will be used.",
+          "type": "integer",
+          "minimum": 3600,
+          "maximum": 7776000,
+          "format": "int64",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "size": {
+          "description": "The size in bytes of the snapshot.",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        },
+        "itemsCount": {
+          "description": "The amount of key-values in the snapshot.",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        },
+        "tags": {
+          "description": "The tags of the snapshot. NOTE: These are data plane tags, not ARM tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
```

Comment:
please rename to reflect that these are dataplane tags

Reply:
this "tags" is a well-defined concept already existing in our api spec